src/registrar/tests/test_views.py

```diff
@@ -1506,6 +1508,30 @@ def test_domain_nameservers_form_submit_missing_host(self):
             status_code=200,
         )
 
+    def test_domain_nameservers_form_submit_duplicate_host(self):
+        """Nameserver form catches error when host is duplicated.
+
+        Uses self.app WebTest because we need to interact with forms.
+        """
+        # initial nameservers page has one server with two ips
+        nameservers_page = self.app.get(reverse("domain-dns-nameservers", kwargs={"pk": self.domain.id}))
+        session_id = self.app.cookies[settings.SESSION_COOKIE_NAME]
+        self.app.set_cookie(settings.SESSION_COOKIE_NAME, session_id)
+        # attempt to submit the form with duplicate host names of fake.host.com
+        nameservers_page.form["form-0-ip"] = ""
+        nameservers_page.form["form-1-server"] = "fake.host.com"
```

Comment:
**question (non-blocking)**: Where does this magic value of `fake.host.com` come from? It looks from the code like it is already in one of these form fields. Just for ease of understanding the test, could we just fill the same server name into `form-1-server` and `form-0-server` so we don't have to know this?